sr_unix/gtmxc_types.h

```diff
@@ -10,103 +10,61 @@
  *								*
  ****************************************************************/
 
-/*	gtmxc_types.h - GT.M, Unix Edition External Call type definitions.  */
+/* The current version of this header file is now called libyottadb.h. This header file contains only depcrecated
+ * forms (and is in fact deprecated itself on YottaDB) but maintains the ability of GT.M installation conversions
```

Comment:
I did not understand the "deprecated itself on YottaDB" part.

Reply:
The gtmxc_types.h header file is not needed for an application utilizing only the YDB provided interfaces (and is thus deprecated ). The gtmxc_types.h header file is only needed for compatibility with the deprecated GTM style interfaces. Actually calling them deprecated instead of just "GTM compatible definitions" may be confusing and even incorrect so I'll reword this a bit to remove most references to "deprecated".